### ISSUE TITLE
Resolution errors when running under typescript

### ISSUE THREAD

**Issue by megakoresh** [jquense/react-widgets#971]
With the following webpack.config:

```typescript
import * as path from 'path';
import HtmlWebpackPlugin = require('html-webpack-plugin');
import * as webpack from 'webpack';

const SRC_PATH = path.join(__dirname);

const webpackConfig: webpack.Configuration = {
  mode: 'development',
  devtool: 'source-map',
  context: SRC_PATH,
  entry: {
    userportal: [
      'react-hot-loader/patch',
      './App.tsx',
    ],
  },
  resolve: {
    extensions: ['.js', '.jsx', '.tsx'],
    modules: [
      SRC_PATH,
      path.resolve(__dirname, 'node_modules'),
    ],
  },
  output: {
    path: __dirname,
    publicPath: '/',
    filename: 'app-[name]-[hash].js',
  },
  watchOptions: {
    ignored: /node_modules/,
  },
  module: {
    rules: [
      {
        test: /\.(j|t)s(x?)$/,
        exclude: /node_modules/,
        use: [
          {
            loader: 'ts-loader',
          },
        ],
      },
      {
        enforce: 'pre',
        test: /\.js$/,
        loader: 'source-map-loader',
      },
      {
        test: /\.css$/,
        use: [
          'style-loader',
          'css-loader',
        ],
      },
      {
        test: /\.less$/,
        use: [
          'style-loader',
          'css-loader',
          'less-loader',
        ],
      },
      {
        test: /\.scss$/,
        exclude: /node_modules/,
        use: [
          'style-loader',
          {
            loader: 'css-loader',
            options: {
              importLoaders: 2,
            },
          },
          'postcss-loader',
          'sass-loader',
        ],
      },
      {
        test: /\.gif$/,
        loader: 'url-loader?mimetype=image/png',
      },
      {
        test: /\.woff(2)?(\?v=[0-9].[0-9].[0-9])?$/,
        loader: 'url-loader?mimetype=application/font-woff',
      },
      {
        test: /\.(ttf|eot|svg)(\?v=[0-9].[0-9].[0-9])?$/,
        loader: 'file-loader?name=[name].[ext]',
      },
      {
        test: /\.(png|jpg|jpeg)$/,
        exclude: /node_modules/,
        use: [
          {
            loader: 'file-loader',
            options: {
              name: 'images/[name].[ext]',
            },
          },
        ],
      },
    ],
  },
  plugins: [
    new HtmlWebpackPlugin({
      chunks: ['userportal'],
      template: './index.ejs',
      filename: 'index.html',
    }),
    new webpack.HotModuleReplacementPlugin(),
    new webpack.NoEmitOnErrorsPlugin(),
    new webpack.DefinePlugin({
      'process.env': {
        NODE_ENV: JSON.stringify('development'),
        GA_TRACKING_ID: JSON.stringify('UA-127718999-1'),
        TEMPLATE_EXPIRATION_INTERVAL: 0,
      },
    }),
  ],
  externals: {
    'react': 'React',
    'react-dom': 'ReactDOM',
  },
};

export default webpackConfig;
```

Produces these errors:

```
ERROR in ../node_modules/react-widgets/lib/NumberInput.js
Module not found: Error: Can't resolve 'dom-helpers/util/inDOM' in '/Users/megak/at-ui/node_modules/react-widgets/lib'
 @ ../node_modules/react-widgets/lib/NumberInput.js 6:36-69
 @ ../node_modules/react-widgets/lib/NumberPicker.js
 @ ../node_modules/react-widgets/lib/index.js
 @ ./common/components/FormDropdownList.tsx
 @ ./common/components/index.js
 @ ./auth/LoginPage.jsx
 @ ./sites/UserPortalApp.jsx
 @ ./sites/userportal.jsx
 @ multi react-hot-loader/patch ./sites/userportal.jsx

ERROR in ../node_modules/react-widgets/lib/util/scrollManager.js
Module not found: Error: Can't resolve 'dom-helpers/util/scrollTo' in '/Users/megak/at-ui/node_modules/react-widgets/lib/util'
 @ ../node_modules/react-widgets/lib/util/scrollManager.js 6:39-75
 @ ../node_modules/react-widgets/lib/DateTimePicker.js
 @ ../node_modules/react-widgets/lib/index.js
 @ ./common/components/FormDropdownList.tsx
 @ ./common/components/index.js
 @ ./auth/LoginPage.jsx
 @ ./sites/UserPortalApp.jsx
 @ ./sites/userportal.jsx
 @ multi react-hot-loader/patch ./sites/userportal.jsx

...+ a ton more of these things, all from dom-helpers package
```

When compiled with this `tsconfig.json`:

```json
{
  "compilerOptions": {
    "outDir": "./dist/",
    "jsx": "react",
    "module": "commonjs",
    "declaration": false,
    "noImplicitAny": false,
    "removeComments": true,
    "noLib": false,
    "emitDecoratorMetadata": true,
    "experimentalDecorators": true,
    "esModuleInterop": true,
    "target": "es2015",
    "lib": [
      "es2015",
      "dom"
    ],
    "incremental": true,
    "sourceMap": true,
    "allowJs": true,
    "strict": true,
    "noImplicitReturns": true,
    "noUnusedLocals": true
  },
  "include": [
    "src/**/*.ts",
    "src/**/*.jsx",
    "src/**/*.tsx",
    "src/**/*.js",
    "server/*.ts"
  ],
  "exclude": [
    "node_modules"
  ]
}
```

And launched via command:

```sh
NODE_ENV=development nodemon -r ts-node/register -w server ./index.ts
```

Minimal `App.tsx` to repro:

```tsx
import React from 'react';

import 'react-widgets/lib/less/react-widgets.less';
import { DropdownList } from 'react-widgets';
import ReactDOM from 'react-dom';

ReactDOM.render(
  <DropdownList />,
  document.getElementById('app')
);
```

I have stripped babel from this configuration, but the same errors were there with it too. 

I don't see any other explanation other than incorrectly generated module resolution statements. Perhaps there is something with lerna-generated files that interferes with typescript - I am not familiar with it to give any concrete pointers, but the issue is 100% reproducible with the latest published version of the package and the source provided above.

The dev server uses express and `webpack-dev-middleware` as well as `webpack-hot-middleware`. Can provide source as well if needed.

If there's some esoteric setting I have to configure to make it work, please let me know - in that case the fix needs to be in documentation for the library.

**Comment by megakoresh** [comment]
PS: I ran against latest available version `5.0.0-beta.0` - same issue.

**Comment by Zakir289** [comment]
Any update on this?
I am also facing the same issue
